# ARBE pointcloud format.

uint32 id               # ID
float32 x               # X axis, unit:m
float32 y               # Y axis, unit:m
float32 z               # Z axis, unit:m
float32 rcs             # reflectivity, 0~255
float32 doppler         # livox tag


